_includes/specials/taproot/en/12-backups.md

```diff
@@ -0,0 +1,101 @@
+  with only one of the original keys.  (That exercise also uses backup
+  keys, but we'll cover that separately in the next point.)  Tweaking
+  the time parameters and the key settings provides you with a flexible
+  and powerful backup construct.
+
+    For example, imagine you can normally spend using a combination of
+    your laptop, mobile phone, and a hardware signing device.  If one of
+    those becomes unavailable, you can wait a month to be able to spend
+    with the remaining two devices.  If two devices become unavailable,
+    you can spend using just one after six months.
+
+    In the normal case of using all three devices, your onchain script
+    is maximally efficient and private.  In the other cases, it's a bit
+    less efficient but may still be reasonably private (your script and
+    its tree depth will look similar to the scripts and depths used in
+    many other contracts).
+
+- **Social recovery for backups and security:** the example above is
+  great at protecting you if one of your devices gets stolen by an
+  attacker, but what happens if two of your devices are stolen?  Also,
+  if you frequently use your wallet, do you really want to wait even a
+  month before you can start spending again after losing a device?
+
+    Taproot makes it easy, cheap, and private to add a social element to
+    your backups.  In addition to the scripts in the previous example,
+    you can also allow immediate spending of your bitcoins with two of
+    your devices plus signatures from two of your friends or family
+    members.  Or immediate spending with only a single one of your keys
+    plus signatures from five people you trust.  (A similar non-social
+    version of this would simply be using extra devices or seed phrases
+    you have stored in secure locations.)
+
+- **Combining time and social thresholds for inheritance:** combining
+  the techniques above, you can allow someone or a group of people to
+  recover your funds in case you suddenly die or become incapacitated.
+  For example, if you haven't moved your bitcoins for six months, you
+  can allow your lawyer and any three of your five most trusted
+  relatives to spend your coins.  If you normally move your bitcoins
+  every six months anyway, this inheritance preparation has no added
+  onchain cost for as long as you live and is completely private from
+  outside observers.  You can even keep the transactions you make
+  private from your lawyer and family as long as you have a reliable way
+  for them to learn your wallet's extended public key (xpub) after your
+  death.
+
+    Please note that making it possible for your heirs' to spend your
+    bitcoins doesn't mean that they'll be able to spend those coins
+    legally.  We recommend that anyone planning to pass on Bitcoins read
+    *Cryptoasset Inheritance Planning* by Pamela Morgan ([physical book
+    and DRM'd ebook][cip amazon] or [DRM-free ebook][cip aantonop]) and
```

Comment:
I think it's fine for us to recommend this if @harding thinks it's a good resource. My only question is about how specific this book is to US law. We have a very international readership, so we should highlight who this book would be useful for.

Reply:
Nah, not specific.  I just started to re-skim it to make sure, but the intro answered the question better than I could've:

![2021-09-14-06_41_20_271403483](https://user-images.githubusercontent.com/61096/133298972-b020ea08-c352-4422-854b-8d75197d3be7.png)

(Also, FWIW, inheritance law in the U.S. is a reserved power, so every state has its own laws.  A book specific to the U.S. would still need fifty different chapters, and I bet some of them could be pretty weird (e.g. Louisiana's law is based on the Napoleonic Code, not U.K. common law).)

And, yeah, I do think it's a good resource.

Reply:
Great. Thanks Dave!